app/controllers/api/account.php

```diff
@@ -499,7 +506,7 @@
                         'email' => $email,
                         'emailVerification' => true,
                         'status' => true, // Email should already be authenticated by OAuth2 provider
-                        'password' => Auth::passwordHash(Auth::passwordGenerator()),
+                        'password' => Auth::passwordHash(Auth::passwordGenerator(), 'bcrypt'),
```

Comment:
Shouldn't we use `scrypt` as the new default?

Reply:
We have been using `bcrypt` until now, so to prevent hell during migration, I would stick with current as default.

Reply:
I think we should support the latest algorithm, we should add a hash attribute to each user with bcrypt for every user. It's not much different from things we have done before on migration. After migration each new user created will use the best and latest algorithm.

Reply:
From the feedback I got, the safest algo at the moment seems to be `Argon2`, I wrote an implementation for it (we should support it anyway for importing purposes).

Please let me know and we can start using it as the default hashing algo.